src/Navbar.tsx

```diff
@@ -16,7 +17,7 @@ type SynapseSettingLink = {
   settingSubPath?: string
 }
 
-class Navbar extends React.Component {
+class Navbar extends React.Component<any, any> {
```

Comment:
could we use a type here for the state instead of `any`, e.g.
```
type State = {
  showMenu: boolean
}
class Navbar extends React.Component<any, State> {...
```

Reply:
Good idea! Code updated.